login.js

```diff
@@ -0,0 +1,106 @@
+const express = require('express');
+const mysql = require('mysql');
+const bcrypt = require('bcrypt');
+const cors = require('cors');
+const app = require('./app');
+
+const PORT = 7777;
+
+const con = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: 'kvnmaria@12',
```

Comment:
Please use a .env filw to load configurations

https://www.npmjs.com/package/dotenv